src/components/PeopleTable/PeopleTable.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import Person from '../person/Person';
+import Sort from '../sort/Sort';
+
+
+
+class PeopleTable extends React.Component {
+  render(){
+    const {
+      selectPerson,
+      inputValueChange,
+      inputValue,
+      sortTypeChange,
+      filterSortList,
+    } = this.props;
+
+    return (
+      <>
+        <Sort  sortTypeChange={sortTypeChange}/>
+        <div className='filter'>
+          <input type='text' value={inputValue} placeholder="Write filter text" onChange={(event) => inputValueChange(event)} />
```

Comment:
event - не потрібно писати, він там буде все рівно
```suggestion
          <input type='text' value={inputValue} placeholder="Write filter text" onChange={() => inputValueChange()} />
```